### ISSUE TITLE
Splash page example styling

### ISSUE THREAD

**Issue by nsheff** [pepkit/pephub#28]
It's strange to me that the heading styles are smaller and less pronounced than the items underneath them:

![image](https://user-images.githubusercontent.com/5271009/166560444-489dc8eb-d738-4276-9c6c-cf25111a5ebf.png)

See how the "GET /pep/demo" is larger than the header class explaining what that is? To my eye that makes it hard to parse at a glance what is going on here. maybe it's just me.

**Comment by nleroy917** [comment]
No, I think you are right -- It seems small at glance now. It's an `<h6/>` tag -- maybe it can be changed to `<h2/>` or `<h3/>`.

**Comment by nleroy917** [comment]
<img width="1191" alt="image" src="https://user-images.githubusercontent.com/41063083/167891678-8aaefd28-2b11-4a00-b2b6-51fa87b7c062.png">

Does this seem more appropriate?

**Comment by nsheff** [comment]
> Does this seem more appropriate?

Yes, that looks much better!
